pkg/localstore/localstore.go

```diff
@@ -214,7 +215,8 @@ func New(path string, baseKey []byte, o *Options, logger logging.Logger) (db *DB
 	}
 
 	// Index storing actual chunk address, data and bin id.
-	db.retrievalDataIndex, err = db.shed.NewIndex("Address->StoreTimestamp|BinID|Data", shed.IndexFuncs{
+	headerSize := 16 + postage.StampSize
+	db.retrievalDataIndex, err = db.shed.NewIndex("Address->StoreTimestamp|BinID|BatchID|Sig|Data", shed.IndexFuncs{
```

Comment:
We will indeed need a migration (or at least a policy on how to deal with this), but since some minor details might still change in localstore we will add this to the list of tasks that need be addressed before this goes to `master` :+1: